scipy/special/tests/test_cdflib.py

```diff
@@ -691,7 +693,7 @@ def test_nctdtr_gh8344(self):
 
     @pytest.mark.parametrize(
         "df, nc, x, expected, rtol",
-        [[3., 5., -2., 1.5645373999149622e-09, 5e-9],
+        [[3., 5., -2., 1.5645373999149622e-09, 2e-8],
```

Comment:
It'd be nice to add a comment here saying that this value can be changed back to 5e-9 after we upgrade to boost-math 1.90.0